### DISCUSSION TITLE
[Unstable] Broken electrical devices underwater produce both smoke and bubble particles

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#15323] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Exactly what the title says. While testing a submarine in mission mode on Unstable last night, I noticed that the broken junction boxes and supercapacitors in a flooded room were both smoking and producing bubbles at the same time. Seems to happen consistently.
I assume the intention is that they should only produce bubbles when underwater?

### Reproduction steps

1. Host a multiplayer server from the main menu.
2. Start a mission on any submarine.
3. Allow the submarine to become flooded and wait for the electrical devices to break.
4. Observe two different kinds of particles appear around the devices at the same time.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This doesn't seem to be a new issue in unstable. Afaik they've always emitted smoke when submerged too.

**Reply by NotWendy** [reply]
Guess I never paid close attention to the particles until now.
